backend/src/schema/resolvers/reports.js

```diff
@@ -23,7 +23,7 @@ export default {
             MATCH (resource {id: $resourceId})
             WHERE resource:User OR resource:Post OR resource:Comment
             MERGE (resource)<-[:BELONGS_TO]-(report:Report {closed: false})
-            ON CREATE SET report.id = randomUUID(), report.createdAt = $createdAt, report.updatedAt = report.createdAt, report.rule = 'latestReviewUpdatedAtRules', report.disable = resource.disabled, report.closed = false
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-02-12T17:16:40Z" title="Wednesday, February 12th 2020, 6:16:40 pm +01:00">Feb 12, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -23,7 +23,7 @@ export default {
             MATCH (resource {id: $resourceId})
             WHERE resource:User OR resource:Post OR resource:Comment
             MERGE (resource)<-[:BELONGS_TO]-(report:Report {closed: false})
-            ON CREATE SET report.id = randomUUID(), report.createdAt = $createdAt, report.updatedAt = report.createdAt, report.rule = 'latestReviewUpdatedAtRules', report.disable = resource.disabled, report.closed = false
```


That the final decision is not stored in the reporting `disable` finds my greatest disagreement @roschaefer !!!

This should never have been refactored out!
You never should recalculate decisions. Something like this always has to be stored at the moment where it  happens.

Think of a judge would come to you and say you have to go in prison, because we have on records that you was guilty on a case and we found out that you never was in prison.
You may say: Sorry I was spoken free at that time.
And they say: We have recalculated it and the outcome is that you was guilty. So come and get your arrest.

That would be funny, or?